pkg/controller/network/nsnetworkpolicy/nsnetworkpolicy_controller.go

```diff
@@ -573,6 +573,13 @@ func (c *NSNetworkPolicyController) processNSNPWorkItem() bool {
 	return true
 }
 
+func workspaceNetworkIsolationEnabled(wksp *workspacev1alpha1.Workspace) bool {
```

Comment:
Please add some unit test for `workspaceNetworkIsolationEnabled` and `namespaceNetworkIsolateEnabled`